CHANGELOG.md

```diff
@@ -26,6 +28,12 @@
 - The "Submit" button in the admin bar updates properly to "Submitted" if the submission happens in the page settings modal.
 - Skipping positional arguments in fragments now works as expected.
 
+### Changes
+- Editor modal's Save button (publish / save draft / submit) now updated to use the `AposSplitButton` component. 
+Editors can choose from several follow-up actions that occur after save, including creating another piece of content 
```

Comment:
These should either have bullets or remove the line breaks. I'd probably say to remove the line breaks.